doc/api/api.md

```diff
@@ -0,0 +1,32 @@
+[![Sedona](../logo.png)](/)
```

Comment:
Yes, this can be ignored.
I only copied it from autogenerated `build/doc` so that I could build from root `doc` folder.
Also, as the old html doc also had one, I left it there for consistency.